geomesa-hbase/pom.xml

```diff
@@ -32,6 +32,11 @@
                 <artifactId>geomesa-hbase-datastore_${scala.binary.version}</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>12.0.1</version>
```

Comment:
i think we already excluded it from the shaded jar, so we might not need this any more. If we keep it we should use the variable `${hbase.guava.version}`